README.md

```diff
@@ -45,6 +45,25 @@ to your Prometheus server, and the files in `dashboards_out` need to be imported
 into you Grafana server.  The exact details will depending on how you deploy your
 monitoring stack to Kubernetes.
 
+### Dashboards for Windows Nodes
+There are separate dashboards for windows resources.
+1) Compute Resources / Cluster(Windows)
+2) Compute Resources / Namespace(Windows)
+3) Compute Resources / Pod(Windows)
+4) USE Method / Cluster(Windows)
+5) USE Method / Node(Windows)
+
+These dashboards are based on metrics populated by wmi_exporter(https://github.com/martinlindhe/wmi_exporter) from each Windows node.
+
+Steps to configure wmi_exporter
+1) Download the latest version of wmi_exporter from release page.(TODO: PR for fetching container metrics is not merge yet(https://github.com/martinlindhe/wmi_exporter/pull/320), Will Update the correct version here once new version is release with container metrics changes.)
```

Comment:
I did not realize this. I would prefer to hold off on merging this pull request until these metrics are available in a release of the wmi_exporter.

Reply:
Container metrics changes got merged and latest wmi_exporter release contains the container metrics as well.
I have updated the README with the details.